src/Pokemon.java

```diff
@@ -0,0 +1,51 @@
+public abstract class Pokemon {
+
+    //Variables
+    private final String name;
+    private int level;
+    private final int hp;
```

Comment:
final op hp is niet helemaal realistisch aangezien een pokemon tijdens een gevecht hp kan kwijt raken.